substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/classinitialization/ClassInitializationFeature.java

```diff
@@ -295,6 +299,33 @@ private void reportMethodInitializationInfo(String path) {
         }
     }
 
+    private static void reportTrackedClassInitializationTraces(String path) {
+        Map<Class<?>, StackTraceElement[]> initializedClasses = ConfigurableClassInitialization.getInitializedClasses();
+        int size = initializedClasses.size();
+        if (size > 0) {
+            // Print to file if required
+            if (Options.PrintClassInitialization.getValue()) {
```

Comment:
Thinking about it more now, we could probably use Options.TraceClassInitialization.hasBeenSet() here instead. I think that would make it more usable as it wouldn't require two different options to be found and set.

Reply:
`TraceClassInitialization` option has been already checked before entering this `reportTrackedClassInitializationTraces` method. I checked `PrintClassInitialization` option here to print an additional message into file besides console.

If we don't need to print the trace at two places but only to file, this check can be simply removed together with console printing code below.

Reply:
That's a good point - I think we should only print the traces to a file and remove the leftover code. I'm of course open to discussion about that, though I think that would be the most uniform approach

Reply:
fixed